src/graphql/schema/Schema.js

```diff
@@ -0,0 +1,226 @@
+const {
+	GraphQLList, GraphQLObjectType, GraphQLSchema, GraphQLString, GraphQLInt, GraphQLInputObjectType
+} = require('graphql');
+const axios = require('axios');
+/*const {GraphQLInputObjectType} = require('graphql/type/definition');*/
+const _ = require('lodash');
+const PermissionsHelper = require('./helpers/PermissionsHelper');
+const UTIL = require('./utils/Utils');
+const {baseUrl} = require('../GraphQLConfig');
+
+const params = {
+	jwt: null,
+	permissionsGroupOld: [],
+	permissionsSystem: [],
+};
+const optionsAxios = ((method, url) =>
+		UTIL.optionsAxios(method, url, params.jwt)
+);
+const setJWT = ((context) =>
+		params.jwt = UTIL.setJWT(context)
+);
+const ADMIN_GROUP_NAME = 'admin';
+
+const PermissionTypeOutput = new GraphQLObjectType({
+	name: 'PermissionOut',
+	fields: {
+		subject: {type: GraphQLString},
+		actions: {type: new GraphQLList(GraphQLString)},
+	},
+});
+
+const PermissionTypeInput = new GraphQLInputObjectType({
+	name: 'PermissionIn',
+	fields: {
+		subject: {type: GraphQLString},
+		actions: {type: new GraphQLList(GraphQLString)},
+	},
+});
+
+const UserType = new GraphQLObjectType({
+	name: 'User',
+	fields: () => ({
+		username: {type: GraphQLString},
+		id: {type: GraphQLString},
+		profile: {type: GraphQLString},
+		permissions: {
+			type: new GraphQLList(PermissionTypeOutput),
+			resolve: async (parentValue) => {
+				if (parentValue.profile !== ADMIN_GROUP_NAME) {
+					return await axios(optionsAxios(UTIL.GET, `/auth/pap/group/${parentValue.profile}/permissions`)).then(res => PermissionsHelper.parsePermissionsFromAuthBack(res.data.permissions)).catch(e => {
+						console.log(e);
+					});
+				}
+				return PermissionsHelper.parsePossiblesPermissionsFromJson();
+			},
+		},
+	}),
+});
+
+const LoginType = new GraphQLObjectType({
+	name: 'Login',
+	fields: () => ({
+		jwt: {type: GraphQLString},
+		user: {
+			type: UserType,
+			resolve(parentValue) {
+				return {
+					username: parentValue.user.username,
+					id: parentValue.user.userid,
+					profile: parentValue.user.profile,
+				};
+			},
+		},
+	}),
+});
+
+const RootQuery = new GraphQLObjectType({
+	name: 'RootQueryType',
+	fields: {
+		user: {
+			type: LoginType,
+		},
+	},
+});
+const ResponseOutput = new GraphQLObjectType({
+	name: 'Response',
+	fields: {
+		message: {type: GraphQLString},
+		status: {type: GraphQLInt},
+		subject: {type: GraphQLString},
+		action: {type: GraphQLString},
+	},
+});
+
+function axiosPermissionsSystem() {
+	return axios(optionsAxios(UTIL.GET, '/auth/pap/permission?type=system')).then((res) => {
+		params.permissionsSystem = res.data.permissions;
+	});
+}
+
+function axiosPermissionsGroup(groupName) {
+	return axios(optionsAxios(UTIL.GET, `/auth/pap/group/${groupName}/permissions`)).then((res) => {
+		params.permissionsGroupOld = res.data.permissions;
+	});
+}
+
+const RootQuery2 = new GraphQLObjectType({
```

Comment:
Does it need exactly this function name? If not, i think the name can be something like `parsePermission` (only suggestions)